lib/Importer/ONNXModelLoader.cpp

```diff
@@ -512,6 +512,13 @@ bool ONNXModelLoader::loadOperator(const ONNX_NAMESPACE::NodeProto &op) {
     return true;
   }
 
+  if (typeName == "LengthsToRanges") {
+    auto in = getNodeValueOrCreateConstantByName(op.input(0));
+    auto *node = G_.createLengthsToRanges(opName, in);
+    addNodeAsOutput(op, node);
+    return true;
+  }
+
```

Comment:
This "LengthsToRanges" op can be added in CommonOperatorLoader.h. 
Btw, I can't find this op in ONNX op list : https://github.com/onnx/onnx/blob/master/docs/Operators.md.
Is it some customized op in ONNX?

Reply:
Yes.

https://github.com/pytorch/pytorch/blob/d4c58216d79f1e1b263e8eeb24cc7a466f3cbc55/caffe2/onnx/torch_ops/defs.cc#L8-L108

Reply:
@yinghai I tried to search "LengthsToRanges" in the webpage but failed. Is there anything I missed? Thanks!

Reply:
We will add that.